models/junos/security_policies_global/junos_security_policies_global.yml

```diff
@@ -0,0 +1,211 @@
+        description:
+          - Specifies the policy action that has to be taken when the packet matches the criteria.
+        type: dict
+        suboptions:
+          name:
+            description:
+              - Name of the stateful firewall rule set
+          stateful_firewall_rule:
+            description:
+              - Specifies the policy action that has to be taken when the packet matches the criteria.
+            type: dict
+            suboptions: 
+              name:
+                description:
+                  - Name of the stateful firewall rule
+                type: str
+      traceoptions:
+        description: A dictionary of security policies
+        type: dict
+        suboptions:
+          file:
+            description: A dictionary to configure the trace file options
+            type: dict
+            required: true
+            suboptions:
+              filename:
+                description: 
+                  - Name of the file to receive the output of the tracing operation.
+                type: str
+                required: true
+              files:
+                description: 
+                  - Maximum number of trace files
+                type: int
+              match:
+                description: Refine the output to include lines that contain the regular expression.
+                type: str
+              size:
+                description: 
+                  - Maximum size of each trace file, in kilobytes (KB), megabytes (MB),or gigabytes (GB)
+                type: dict
+                suboptions:
+                  amount:
+                    description: The numerical value of the max file size
+                    type: int
+                  unit:
+                    description: The unit of size; K to specify KB, m to specify MB, or g to specify GB
+                    choices:
+                      - K
+                      - m
+                      - g
+              world_readable:
+                description: The world_readable option enables any user to read the file.
+                type: bool
+              no_world_readable:
+                description: Log files can be accessed only by the user who configures the tracing operation.
+                type: bool
+          flag:
+            description:
+              - Trace operation to perform.
+            choices:
+              - all
+              - configuration
+              - compilation
+              - ipc
+              - lookup
+              - routing_socket
+              - rules
+          no_remote_trace:
+            description: Set remote tracing as disabled.
+            type: bool
+      unified_policy:
+        description:
+          - Configure maximum number of policy lookups, when micro-applications reach the transaction final state, during the dynamic application classification process.
+        type: dict
+        suboptions:
+          max_lookups:
```

Comment:
Similar comment as earlier, can't the unified policy be individual param as: `unified_policy_max_lookups`